packages/edit-site/src/components/global-styles/font-library-modal/upload-fonts.js

```diff
@@ -230,8 +230,7 @@ function UploadFonts() {
 						onChange={ onFilesUpload }
 						render={ ( { openFileDialog } ) => (
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> styles --> typography --> open font library modal --> upload font tab.

**Note:** Due to higher specificity of button class, it has changed the size of upload button to 40px which is not looking nice. Should we revert this change or add some style change by adding !importnant or higher specificity? Cc: @mirka @ciampo 

| Before | After     |
| :-------- | :------- |
| <img width="625" alt="upload-font-before" src="https://github.com/user-attachments/assets/d8383997-0c0b-4370-a858-cf2db21311ff"> | <img width="625" alt="Upload-font-after" src="https://github.com/user-attachments/assets/07e5bac4-7b0f-4e52-b629-350d53b75989"> |

Reply:
I raised this as a separate issue at #65333, but for the scope of this PR let's just add an `!important` to the height.

Reply:
> I raised this as a separate issue at #65333, but for the scope of this PR let's just add an `!important` to the height.

Sure 👍.